src/core/api/nodopagamenti_api/nodoPerPM/v2/_swagger.json.tpl

```diff
@@ -26,350 +21,475 @@
+                "example": "2022-02-22T14:41:58"
+              },
+              "rrn": {
+                "type": "string",
+                "description": "Transaction identifier on the card circuit"
+              },
+              "authorizationCode": {
+                "type": "string",
+                "description": "Authorization code sent by the payment gateway, present only if `outcomePaymentGateway` is `OK`"
+              }
+            }
           }
         },
-        "outcome": {
-          "type": "string",
-          "enum": [
-            "OK",
-            "KO"
-          ]
-        },
-        "idPSP": {
-          "description": "required only for outcomePaymentGateway OK",
-          "type": "string",
-          "minLength": 1,
-          "maxLength": 35
-        },
-        "idBrokerPSP": {
-          "description": "required only for outcomePaymentGateway OK",
-          "type": "string",
-          "minLength": 1,
-          "maxLength": 35
-        },
-        "idChannel": {
-          "description": "required only for outcomePaymentGateway OK",
-          "type": "string",
-          "minLength": 1,
-          "maxLength": 35
-        },
-        "paymentMethod": {
-          "description": "required only for outcomePaymentGateway OK",
-          "type": "string",
-          "minLength": 1
-        },
-        "transactionId": {
-          "type": "string",
-          "minLength": 1,
-          "maxLength": 255
-        },
-        "totalAmount": {
-          "description": "required only for outcomePaymentGateway OK",
-          "type": "number",
-          "minimum": 0,
-          "maximum": 1000000000,
-          "example": "20.10"
-        },
-        "fee": {
-          "type": "number",
-          "description": "required only for outcomePaymentGateway OK",
-          "minimum": 0,
-          "maximum": 1000000000,
-          "example": "10.00"
-        },
-        "timestampOperation": {
-          "description": "required only for outcomePaymentGateway OK",
-          "type": "string",
-          "format": "date-time",
-          "example": "2022-02-22T14:41:58.811+01:00"
-        },
-        "additionalPaymentInformations": {
-          "$ref": "#/definitions/AdditionalPaymentInformations"
+        "required": [
+          "paymentMethod",
+          "additionalPaymentInformations"
+        ]
+      },
+      "RedirectAdditionalPaymentInformations": {
+        "type": "object",
+        "properties": {
+          "paymentMethod": {
+            "type": "string",
+            "description": "Redirect payment method name"
+          },
+          "additionalPaymentInformations": {
+            "type": "object",
+            "description": "Additional payment data pertaining to Redirect-type payments",
+            "properties": {
+              "tipoVersamento": {
+                "type": "string",
+                "description": "Specific Checkout Redirect gateway"
+              },
```

Comment:
are you sure this field has to be added for redirect close payments? why is in italian?

Reply:
Copy-paste gone wrong, removing it! Thanks 👍

Reply:
Fixed with 0f17d1c5